### PR TITLE
put dashboard in kubeaddons

### PR DESCRIPTION
**Pull request by jr0d**

### CODE REVIEW

**Comment by jaxesn** [comment]
Will there be an issue with upgrading with it changing namespaces? If so, I'm fine if that isn't handled here initially just something we need to handle before 1.3 ga.

**Comment by jr0d** [comment]
I am pretty sure helm will handle the upgrade case.. resources are all namespaced and the pod will get launched with the new argument.

**Comment by shaneutt** [comment]
@jr0d in this interim before we have the new repository set up completely, please make a copy of this PR against https://github.com/mesosphere/kubernetes-base-addons, this wont be required for much longer once we fully deprecate this repo.

File: templates/kommander.yaml [hunk 1]
```diff
@@ -29,8 +29,8 @@ spec:
       enabled: true
   chartReference:
     chart: kommander
-    repo: https://mesosphere.github.io/charts/stable
-    version: 0.2.27
+    repo: http://blacknode.net:8080/
```
Comment (hectorj2f) [inline templates/kommander.yaml]: this needs to be changed before merging it

**Comment by jr0d** [comment]
https://github.com/mesosphere/kubernetes-base-addons/pull/22

**Comment by jr0d** [comment]
@shaneutt I've created the PR for kubernetes-base-addons

**Comment by hectorj2f** [comment]
@jr0d did you have the chance to test it on a konvoy cluster ?

**Comment by jr0d** [comment]
@hectorj2f sorry for the late reply, yes, this has been tested.

**Pull request merged**